### DISCUSSION TITLE
pose

### DISCUSSION THREAD

**Discussion by Cryandlonely** [ultralytics/ultralytics#15381] [Q&A]
How to associate the recognized frame with the bone information

**Comment by pderrenger** [comment]
@Cryandlonely to associate the recognized frame with bone information, you can use the `keypoints` attribute in the `Results` object returned by the YOLOv8 model. This attribute contains the detected keypoints for each object, which you can then map to specific bones. If you encounter any issues, ensure you are using the latest version of the package.
